addons/website/static/src/js/content/auto_hide_menu.js

```diff
@@ -7,33 +7,27 @@ import { initAutoMoreMenu } from '@web/legacy/js/core/menu';
  */
 document.addEventListener('DOMContentLoaded', async () => {
     const header = document.querySelector('header#top');
-    if (!header || header.classList.contains('o_no_autohide_menu')) {
-        return;
-    }
-    const topMenu = header.querySelector('#top_menu');
-    const unfoldable = '.divider, .divider ~ li, .o_no_autohide_item';
-    const excludedImagesSelector = '.o_mega_menu, .o_offcanvas_logo_container, .dropdown-menu .o_lang_flag';
-    const excludedImages = [...header.querySelectorAll(excludedImagesSelector)];
-    const images = [...header.querySelectorAll('img')].filter((img) => {
-        excludedImages.forEach(node => {
-            if (node.contains(img)) {
-                return false;
-            }
-        });
-        return img.matches && !img.matches(excludedImagesSelector);
-    });
-    const defs = images.map((image) => {
-        if (image.complete) {
-            return null;
+    if (header) {
+        const topMenu = header.querySelector('#top_menu');
+        if (header.classList.contains('o_no_autohide_menu')) {
+            topMenu.classList.remove('o_menu_loading');
+            return;
```

Comment:
Should not that part be in the parent commit?

Reply:
Actually in the code [here](https://github.com/odoo/odoo/pull/75063/commits/3f2475f6780b5054f371a053055a0dea2dc39856) there is only one update (after all images are loaded) so we remove the `o_menu_loading` at the end, but in the next commit the code will start with a first update then an adjustment (after loading images)...
So if we have a target with `o_no_autohide_menu` we should force the `topMenu.classList.remove('o_menu_loading');` because there is no update that will be executed in this case.